build/lib/Crypto_bot_pkg/crypto_selection.py

```diff
@@ -0,0 +1,17 @@
+class crypto_bot:
+    """
+    A class where you can select your favorite cryptos, track them and use a trading bot    
+    """
+
+    def which_crypto():
+        """
+        This is function 1
+        """
+        print("This function will allow you to choose between BTC, ETH, ADA or DOGE coins")
+        
+    def trade_bot():
+        """
+        This is function 2
```

Comment:
I'd suggest that you briefly explain what the function does in the documentation as well.